scripts/scr_start_load/scr_start_load.gml

```diff
@@ -28,6 +28,7 @@ function scr_start_load(fleet, load_from_star, escort_load) {
 	
 		//loop through companies. try and load whole company onto single ship else spread company across largest ships with remaining space
 	for (var _comp =0; _comp<10;_comp++;){
+		show_debug_message("one")
```

Comment:
is this needed?

Reply:
just a remnant from debugging will remove